config/sync/field.storage.paragraph.su_spacer_size.yml

```diff
@@ -0,0 +1,30 @@
+uuid: 97dd4d2d-9b81-4285-a846-fe1c5fb44d54
+langcode: en
+status: true
+dependencies:
+  module:
+    - options
+    - paragraphs
+id: paragraph.su_spacer_size
+field_name: su_spacer_size
+entity_type: paragraph
+type: list_string
+settings:
+  allowed_values:
+    -
+      value: su_spacer_default
+      label: '-default-'
```

Comment:
Just make the field not required and you don't need this

Reply:
Ok.  I added a note to the Help text that the `- None -` option applies the default size.